cranelift/codegen/src/ir/entities.rs

```diff
@@ -549,6 +549,14 @@ mod tests {
         );
     }
 
+    #[test]
+    fn memory_option() {
```

Comment:
Can remove -- This was mentioned in the doc comments, I added a quick test to ensure this was the case, since there was a test for PackedOption